core/src/main/kotlin/org/evomaster/core/output/naming/RestActionTestCaseNamingStrategy.kt

```diff
@@ -147,9 +147,28 @@ open class RestActionTestCaseNamingStrategy(
         val queryParams = restAction.parameters.filterIsInstance<QueryParam>()
         result.add(with)
         result.add(if (queryParams.size > 1) "${queryParam}s" else queryParam)
+        if (nameWithQueryParameters) {
+            addQueryParameterNames(queryParams, result)
+        }
         return result
     }
 
+    private fun addQueryParameterNames(queryParams: List<QueryParam>, result: MutableList<String>) {
+        val booleanQueryParams = getBooleanQueryParams(queryParams)
+        if (booleanQueryParams.isNotEmpty()) {
```

Comment:
redundant if statement. there is no issue in doing a `forEachIndexed` on a empty collection

Reply:
🤦‍♂️ sorry